sdk/communication/Azure.Communication.CallAutomation/src/CallMedia.cs

```diff
@@ -41,20 +41,18 @@ protected CallMedia()
         }
 
         /// <summary>
-        /// Plays a file async.
+        /// Plays audio to specified participant(s) async.
         /// </summary>
-        /// <param name="playSource"></param>
-        /// <param name="cancellationToken"></param>
-        /// <param name="playTo"></param>
-        /// <param name="playOptions"></param>
-        /// <returns>Returns <see cref="PlayResult"/>, which can be used to wait for Play's related events.</returns>
-        public virtual async Task<Response<PlayResult>> PlayAsync(PlaySource playSource, IEnumerable<CommunicationIdentifier> playTo, PlayOptions playOptions = default, CancellationToken cancellationToken = default)
+        /// <param name="cancellationToken">An optional CancellationToken to cancel the request.</param>
+        /// <param name="options">An optional object containing play options and configurations.</param>
+        /// <returns>Returns a <see cref="PlayResult"/> object, which can be used to wait for Play's related events.</returns>
+        public virtual async Task<Response<PlayResult>> PlayAsync(PlayOptions options, CancellationToken cancellationToken = default)
```

Comment:
is there simple method for Play & PlayToAll?
These 2 methods could be a good candidate for having both simple & complex method - we have complex method with option pattern, but we could also use simple method for better usability.

You can take a look at CreateCall and its overload methods for how it implemented simple & complex methods.